src/app/personside/visittkort/header/status/PersonStatus.tsx

```diff
@@ -0,0 +1,52 @@
+import * as React from 'react';
+import * as moment from 'moment';
+import styled from 'styled-components';
+import Undertekst from 'nav-frontend-typografi/lib/undertekst';
+
+import { BostatusTyper, Person } from '../../../../../models/person';
+
+const emdash = '\u2014';
+const PadLeft = styled.span`
+  margin-left: 2em;
+`;
+
+const NoWrap = styled.span`
+  white-space: nowrap;
+`;
+
+interface PersonProps {
+    person: Person;
+}
+
+function Dødsdato({person}: PersonProps) {
+    if (person.status.dødsdato) {
+        const formatertDødsdato = moment(person.status.dødsdato).format('DD.MM.YYYY');
+        return <>{emdash} Død {formatertDødsdato}</>;
+    } else {
+        return null;
+    }
+}
+
+function Utvandret({person}: PersonProps) {
```

Comment:
trenger kanskje ikke returnere en jsx komponent her? Holder vel å returnere en string..?

Reply:
Alternativet blir kanskje noe sånt som det her:
```
function utvandretTekst(person: Person) {
    if (person.status.bostatus === BostatusTyper.Utvandret) {
        return `${emdash} Utvandret`;
    } else {
        return null;
    }
}

function FødselsnummerLinje({person}: PersonProps) {
    const utvandret = utvandretTekst(person);
    return <>{person.fødselsnummer} <Dødsdato person={person}/>{utvandret}</>;
}
```

Synes det faktisk ble litt cleanere med sånn det var, men vet ikke hva du tenker?

Reply:
Hva med:

```
function utvandret(person: Person) {
    person.status.bostatus === BostatusTyper.Utvandret 
       ? return `${emdash} Utvandret`
       :  return null;
}

function dødsDato({person}: PersonProps) {
    if (person.status.dødsdato) {
        const formatertDødsdato = moment(person.status.dødsdato).format('DD.MM.YYYY');
        return `${emdash} Død {formatertDødsdato}`;
    }
    return null;
}

function fødselsnummerLinje({person}: PersonProps) {
    return {person.fødselsnummer} {dødsDato(person)} {utvandret(person)};
}
```
Da returnerer du bare strings hele veien. Så vidt jeg kan se er det ikke behov for å ha jsx-elementer her?